deeplenstronomy/utils.py

```diff
@@ -145,7 +145,7 @@ def treat_map_like_user_dist(im_dir, size):
     return draw_from_user_dist("unused filename", size, 'iterate', df=df)
     
 
-def draw_from_user_dist(filename, size, mode, step=10, df=None):
+def draw_from_user_dist(filename, size, mode, step=10, df=None, params = None):
```

Comment:
In function definitions, leave no spaces between parameter names, the equal sign, and default values.

Reply:
resolved by 19d8f5b